benchmarks/docker-compose.pgsql-ssl-server-ssl.yml

```diff
@@ -40,7 +39,6 @@ services:
         build:
             context: ../
             dockerfile: docker/postgresql-ssl.dockerfile
-        network_mode: host
```

Comment:
What is the reason for this change? As I know default `network_mode` in compose if `bridge` means that containers will be placed in the compose bridge network and they will not be accessible from the host.  So this removing changes the compose behavior - I am just trying to understand whether it isn`t breaking anything.

Reply:
my docker-compose:
```
$ docker-compose -v
docker-compose version 1.29.2
```
tells me: `docker.errors.InvalidArgument: "host" network_mode is incompatible with port_bindings`
As I think, it is not valid anymore if you declare ports (that we do in this file). I tested locally that ports are accessible from the host and they are.